pkg/kubectl/resource_printer.go

```diff
@@ -786,8 +788,17 @@ func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 			return err
 		}
 	}
+
+	podColor := "<fg 2>" // green
+	if readyContainers != totalContainers {
+		podColor = "<fg 11>" // yellow
+	}
+	if readyContainers == 0 {
+		podColor = "<fg 1>" // red
```

Comment:
I understand the benefit, but we need to reconcile this with https://github.com/kubernetes/community/pull/363